docs/NewOBOFC

```diff
@@ -0,0 +1,19 @@
+---
+layout: doc
+id: OFOCMember
+title: Becoming a member of the OBO Operations Committee
+---
+
+# Becoming a member of the OBO Foundry Operations Committee (OFOC)
+
+Note: This is _not_ a formal process, but rather a piece of documentation to make the _informal_ process more transparent. 
+
+1. If someone (applicant) is interested in becoming a member of the operations group, they should contact an existing group member, 
+who explains to them that the various duties expected from members of the OFOC (e.g. chairing calls, addressing issues on GitHub, joining a working group).
+2. The applicant confirms they are willing to take up some of these duties.
+3. The current members are then asked via an email on obo-operations if anyone objects to adding the candidate member. 
+4. If conflicts/concerns arise, we try to first find consensus during the nearest OFOC call. 
+5. In cases where no consensus can be reached, we ask for a formal vote with the majority of active members
+
+
+Note taht new members are strongly encouraged to  become members of one of the subcommittees: Editorial Working Group or Technical Working Group. 
```

Comment:
taht -> that

WGs: add links to (possibly out of date) pages?
https://obofoundry.org/docs/EditorialWG.html
https://obofoundry.org/docs/TechnicalWG.html